SECURITY.md

```diff
@@ -1,32 +1,56 @@
-# Security
+# Coordinated Vulnerability Disclosure Policy
 
-> **IMPORTANT**: If you find a security issue, you can contact our team directly at
-security@interchain.berlin, or report it to our [bug bounty program](https://hackerone.com/cosmos) on HackerOne. *DO NOT* open a public issue on the repository.
+The Cosmos ecosystem believes that strong security is a blend of highly technical security researchers
+who care about security and the forward progression of the ecosystem and the attentiveness and
+openness of Cosmos core contributors to help continually secure our operations.
 
-## Bug Bounty
+> **IMPORTANT**: *DO NOT* open public issues on this repository for security vulnerabilities.
 
-As part of our [Coordinated Vulnerability Disclosure Policy](https://tendermint.com/security), we operate a
-[bug bounty program](https://hackerone.com/cosmos) with Hacker One.
+## Scope
 
-See the policy linked above for more details on submissions and rewards and read
-this [blog post](https://blog.cosmos.network/bug-bounty-program-for-tendermint-cosmos-833c67693586) for the program scope.
+| Scope                 |
+|-----------------------|
+| last release (tagged) |
+| main branch           |
 
-The following is a list of examples of the kinds of bugs we're most interested
-in for the IBC Golang repository. Please refer to the corresponding repositories for vulnerabilities on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/blob/master/SECURITY.md) and [CometBFT](https://github.com/cometbft/cometbft/blob/master/SECURITY.md) repositories.
+The latest **release tag** of this repository is supported for security updates as well as the **main** branch.
+Security vulnerabilities should be reported if the vulnerability can be reproduced on either one of those.
 
-### IBC Core
+## Reporting a vulnerability
 
-- [`02-client`](https://github.com/cosmos/ibc-go/tree/main/modules/core/02-client)
-- [`03-connection`](https://github.com/cosmos/ibc-go/tree/main/modules/core/03-connection)
-- [`04-channel`](https://github.com/cosmos/ibc-go/tree/main/modules/core/04-channel)
-- [`05-port`](https://github.com/cosmos/ibc-go/tree/main/modules/core/05-port)
-- [`23-commitment`](https://github.com/cosmos/ibc-go/tree/main/modules/core/23-commitment)
-- [`24-host`](https://github.com/cosmos/ibc-go/tree/main/modules/core/24-host)
+| Reporting methods                                                 |
+|-------------------------------------------------------------------|
+| [GitHub Private Vulnerability Reporting](security/advisories/new) |
+| [HackerOne bug bounty program](https://hackerone.com/cosmos)      |
 
-### IBC Applications
+All security vulnerabilities can be reported under GitHub's [Private vulnerability reporting](security/advisories/new) system.
```

Comment:
maybe this could be an absolute reference? `security/advisory/new` is currently a dead-link but that is probably because it is using the branch commit as a base url.

Reply:
No prob @crodriguezvega. there's one more instance of it on line 23 that I see now, missed that too. :smile: